daemon/mgr/container.go

```diff
@@ -473,8 +473,12 @@ func (cm *ContainerManager) stoppedAndRelease(id string, m *ctrd.Message) error
 }
 
 func (cm *ContainerManager) exitedAndRelease(id string, m *ctrd.Message) error {
-	// release io.
 	if io := cm.IOs.Get(id); io != nil {
+		if err := m.StartError(); err != nil {
+			fmt.Fprintf(io.Stdout, "%v\n", err)
```

Comment:
This is will stream log into stdout. But I think it is OK to stream into log, logrus.

If this is Pouchd's log, it is better to use a unified way to collect log, currently `logrus`.

WDYT? @skoo87

Reply:
NO, it is not log,  it's that send the error message to CLI's stdout.

Reply:
Is `io.Stdout` the system stdout of Pouchd?
I am a little confused.

Reply:
No, it's not system stdout,  you can think it's a pipe be used to connect to 'CLI''s STDOUT.

Reply:
Oh, I get it. I think `io` makes me confused. I mistake `io` as `os`.
Could you change name io to be a more readable variable?

Reply:
the said have a reason.  But if you look at the context and understand it, I think you will not confused, believe me the "io" is very readable, ^_^.